isso/views/comments.py

```diff
@@ -954,6 +960,7 @@ def count(self, environ, request, uri):
     @apiSuccessExample Counts of 5 threads:
         [2, 18, 4, 0, 3]
     """
+
```

Comment:
Begone!

Reply:
There's quite a few functions with that inconsistent spacing. Also *technically* the docstrings should be inside the functions, not outside of them. Fixing that feels a bit out-of-scope for this change. Maybe in the future there should be a docstring normalization pass.

Reply:
(That said, I'll remove that accidentally-introduced newline. No idea how that got in there!)

Reply:
Yeah - I think there's value in fixing some of the formatting, but mixing those changes in with other changes just makes everything harder to review.

Reply:
A thing I’ve found to be helpful is to make autopep8 part of the build process so that there’s never an opportunity for things to drift in the first place.

Reply:
Sure, but that should also be a separate PR :)